### DISCUSSION TITLE
New feature from Vehicle Control Addon

### DISCUSSION THREAD

**Discussion by Daentes** [jason0611/FS22_DashboardLive#78] [Ideas]
Hello, is it possible to add a new function that retrieves data from the VCA, showing the status "vcaGearSplitUp / vcaGearSplitDown"?
for example: low group gears / high group gears

**Comment by SbSh-DNA-ModDing** [comment]
Hey.. also the second groupset from realismAddon_Gearbox would be nice, but that's for old machines. It need a animation like a gearlever.
